public/content/developers/docs/design-and-ux/index.md

```diff
@@ -41,18 +40,16 @@ This is a curated list of user research done in web3 that may help with design a
 
 ## Design for web3 {#design-for-web3}
 
+- [Web3 UX Design Handbook](https://web3ux.design/) - Practical guide to designing Web3 apps
 - [Web3 Design Principles](https://medium.com/@lyricalpolymath/web3-design-principles-f21db2f240c1) - A framework of UX rules for blockchain based dapps
 - [Blockchain Design Principles](https://medium.com/design-ibm/blockchain-design-principles-599c5c067b6e) - Lessons learned by the blockchain design team at IBM
-- [Dimensions of web3 UX](https://uxdesign.cc/the-levels-of-web3-user-experience-4f2ad113e37d) - An analysis of different layers of web3 UX and its challenges
```

Comment:
This article is still available. Is the reason for removal because of needing to be a member to access this?